tests/ci/cdk/README.md

```diff
@@ -158,3 +159,57 @@ command.
  * `cdk deploy`      deploy this stack to your default AWS account/region
  * `cdk diff`        compare deployed stack with current state
  * `cdk docs`        open CDK documentation
+ 
+### Useful Docker image build commands
+
+**Notes**:
+* below commands use default values set in `cdk/util/metadata.py`.
+* Always clean up resources set up for Docker image build.
+  * `cdk destroy aws-lc-docker-image-build-* --force`
+
+#### Linux Docker image build
+
+```bash
+# Launch Linux Docker image CodeBuild resources.
+cdk deploy aws-lc-docker-image-build-linux --require-approval never
+
+# Trigger the Linux CodeBuild.
```

Comment:
Maybe: "Trigger CodeBuild to build Linux Docker Images"

Reply:
Changed.